Docker/README.md

```diff
@@ -34,14 +34,14 @@ First usage steps:
 1. Acquire the Docker image. There are two options:
     1. Download the prebuilt images from [ualibraries DockerHub](https://hub.docker.com/r/ualibraries/)
         * `docker pull ualibraries/${image_name}:${optional_tag}`
-    2. Build from `Dockerfile` definition 
+    2. Build from `Dockerfile` definition
         * `docker build -t ualibraries/${image_name}:${optional_tag} -f ${dockerfile} ${dockerfile_directory`
```

Comment:
again, probably a leftover from previous doc: a missing } at the end of the line